idoit-install

```diff
@@ -1510,7 +1510,7 @@ function configureMariaDB {
     mv /var/lib/mysql/ib_logfile[01] "$TMP_DIR" || abort "Unable to remove old log files"
 
     log "How many bytes of your RAM do you like to spend to MariaDB?"
-    echo -n -e "You SHOULD give MariaDB ~ 50 per cent of your RAM [leave empty for '${MARIADB_INNODB_BUFFER_POOL_SIZE}']: "
+    echo -n -e "You SHOULD give MariaDB ~ 50 per cent of your RAM. You can use G for Gigabyte or M for Megabyte as Value e.g. 1024M or 1G [leave empty for '${MARIADB_INNODB_BUFFER_POOL_SIZE}']: "
```

Comment:
Suggestion: `…of your RAM. Use G for Gigabytes or M for Megabytes, e.g. 1024M or 1G [leave…`

Reply:
Thanks, changed that.